src/pytorch_lightning/trainer/connectors/data_connector.py

```diff
@@ -45,6 +45,9 @@
 
 warning_cache = WarningCache()
 
+BATCH_DATALOADER = Union[Collection[DataLoader], List[DataLoader], DataLoader[Any]]
+REQUEST_DATALOADER = Union[DataLoader, List[DataLoader], BATCH_DATALOADER]
+
```

Comment:
```suggestion
```

Reply:
these are needed to resolve errors. each was created based on mypy error output.